saraswati/cli.py

```diff
@@ -85,9 +85,18 @@ def validate_channel(ctx, param, value):
     "-cf",
     "container_format",
     envvar="CONTAINER_FORMAT",
-    type=click.Choice(["matroska", "ogg"]),
+    type=click.Choice(["matroska", "ogg", "none"]),
     default="matroska",
-    help="""Set container format. Use either "matroska" (default) or "ogg".""",
+    help="""Set container format. Use either "matroska" (default) or "ogg" or "none" for no container.""",
+)
+codec_opt = click.option(
+    "--codec-format",
+    "-cf",
+    "codec_format",
+    envvar="CODEC_FORMAT",
+    type=click.Choice(["wav", "flac", "wavpack"]),
+    default="flac",
+    help="""Set codec format. Use either "flac" (default) "wav" or "wavpack".""",
 )
```

Comment:
Let's just call this "`--audio-format`"?

```suggestion
audio_format_opt = click.option(
    "--audio-format",
    "-af",
    "audio_format",
    envvar="AUDIO_FORMAT",
    type=click.Choice(["flac", "wav", "wavpack"]),
    default="flac",
    help="""Set audio format. Use either "flac" (default), "wav" or "wavpack".""",
)
```